.github/workflows/pythonpackage.yml

```
@@ -0,0 +1,179 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install gcc
+      run: |
+        sudo apt-get update
+        sudo apt-get install -yq gcc
+        sudo apt-get install -yq gfortran
+    - name: Install kim-api
+      run: |
+        export KIMPY_DIR=${PWD}
+        export KIM_API_VERSION="2.2.0"
+        cd $KIMPY_DIR && cd ..
+        wget http://s3.openkim.org/kim-api/kim-api-$KIM_API_VERSION.txz
+        tar Jxvf kim-api-$KIM_API_VERSION.txz
+        cd kim-api-$KIM_API_VERSION
+        mkdir build && cd build
+        cmake .. -DCMAKE_BUILD_TYPE=Release
+        make -j2 
+        sudo make install
+        sudo ldconfig
+        cd $KIMPY_DIR
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
```

I do not know the detailed difference between v1 and v2 , but should be better :-)
```suggestion
      uses: actions/setup-python@v2
```

---

@mjwen thanks. updated!